msg/McVelCtrlStatus.msg

```diff
@@ -0,0 +1,6 @@
+uint64 timestamp		# time since system start (microseconds)
+
+# mc velocity controller integrator status
```

Comment:
I would just make this a generic velocity_controller_status message and include everything.

Reply:
The .msg is either generic velocity controller or could even be generic PID controller status, then the TOPIC could be `mc_velocity_controller_status`.